core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -187,6 +188,16 @@ void listStatus(AlluxioURI path, ListStatusContext context, ResultStream<FileInf
   void checkAccess(AlluxioURI path, CheckAccessContext context)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException, IOException;
 
+  /**
+   * Check path exists.
+   *
+   * @param path path to check
+   * @param context the method context
+   * @return whether the path exists
+   */
+  boolean checkExistence(AlluxioURI path, CheckExistenceContext context)
```

Comment:
keep method name in filesystem master consistent with the client-side method to follow existing convention? also for method name in proto and `FileSystemMasterClientServiceHandler`

```suggestion
  boolean exists(AlluxioURI path, CheckExistenceContext context)
```

Reply:
There's no such convention. For example, client side uses `delete` while proto uses `remove`.  `getStatus` on client side vs `getFileInfo` on master.